packages/math-input/src/components/keypad/button-assets.tsx

```diff
@@ -487,6 +487,31 @@ export default function ButtonAsset({id}: Props): React.ReactElement {
             );
 
         default:
-            throw new Error(`Invalid asset ${id}`);
```

Comment:
I know this is already merged, but I'm wondering if we shouldn't have a FallbackSVG component, use that for specific cases that we're expecting, and keep this error for the unexpected.